src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -94,6 +94,80 @@ def provisionInfrastructure(target, item, parameters) {
   }
 }
 
+def __getRbacGroups() {
+  return [ 'developer', 'administrator' ]
+}
+
+def __getClusterRole(group) {
+  def clusterRoleMappings = [
+    developer: 'edit',
+    administrator: 'admin'
+  ]
+  return clusterRoleMappings[group]
+}
+
+def __getRole(namespace, group) {
+  return "${namespace}-${group}".toUpperCase()
```

Comment:
Agreed, consistency is nice. You will need the curly braces if the GString runs straight into characters that are legal in variable names (e.g. underscore instead of hyphen). I'd actually fall on the side of using curly braces everywhere, but that is just personal preference and not a request!

Reply:
I've removed them since none were needed, it's a bad habit I've carried over from bash.